src/components/DefineDissolution/DestroyCertificate.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div>
+    <div v-if="isSummary" class="destroy-certificate-summary">
+      <div v-if="getHasCertificateDestroyed" class="d-flex">
+        <span>
+          <v-icon class="destroy-certificate-success-chk ml-n1 pr-2" color="successCheckmark">mdi-check</v-icon>
+        </span>
+        <span v-html="destroyCertificateDescription" />
```

Comment:
Same as `<span>{{ destroyCertificateDescription }}</span>` i think. 
Just curious the pros and cons to using `v-html`. I could definite see the need if there is some html tags required in the string such as a `<strong>` tag or something. 
Just curious why this approach. Totally no action required, just curious.

Reply:
`v-html` allows you to embed HTML tags in the string.

Reply:
I don't think I did that intentionally